src/int/mod.rs

```diff
@@ -18,6 +18,14 @@ pub mod sdiv;
 pub mod shift;
 pub mod udiv;
 
+#[cfg(not(feature = "expose-alt-impls"))]
+mod leading_zeros;
+
+#[cfg(feature = "expose-alt-impls")]
+pub mod leading_zeros;
```

Comment:
I think it'd be fine to just always export this module, there shouldn't be a need to make it `#[cfg]` pub

Reply:
Making it `pub` always will expose the `usize_leading_zeros...` functions by default, but the only reason they should ever be public is for testing purposes.

Reply:
I think exporting that is fine, there are many exported symbols from this library other than the intrinsics due to traits and such, so we don't need to be super strict about what exactly is exported.

Reply:
I have made it public always and removed unneeded configuration